pkgs/objective_c/pubspec.yaml

```diff
@@ -4,7 +4,7 @@
 
 name: objective_c
 description: 'A library to access Objective C from Flutter that acts as a support library for package:ffigen.'
-version: 2.0.0
+version: 3.0.0-wip
```

Comment:
Why does this need to be a major version bump?

Reply:
Doesn't the addition of any new classes affect the code that should be generated by `package:ffigen`? For example, if we release `package:objective_c` with this change, then existing versions of `package:ffigen` may use it and users may face duplicate symbols: `NSStream` defined in `package:objective_c` and `NSStream` defined by `ffigen`ed code.

Or do I misunderstand how this works?

Reply:
Yeah, I guess we do need a major version bump. That's annoying. I've been treating this sort of thing as a minor version.